packages/integrations/mdx/src/index.ts

```diff
@@ -49,7 +49,7 @@ export default function mdx(mdxOptions: MdxOptions = {}): AstroIntegration {
 						// Workarounds tried:
 						// - "import * as remarkShikiTwoslash"
 						// - "import { default as remarkShikiTwoslash }"
-						(remarkShikiTwoslash as any).default,
+						(remarkShikiTwoslash as any).default ?? remarkShikiTwoslash,
```

Comment:
Good thinking here. Wonder if proload was somehow messing with the `.default` here...

Reply:
That's not why proload was failing. But this is needed to be compatible with loading in Vite.